AbletonMCP_Remote_Script/handlers/browser.py

```
@@ -0,0 +1,693 @@
+        if ctrl is None:
+            raise RuntimeError("get_browser_items_at_path requires ctrl for application()")
+        app = ctrl.application()
+        if not app:
+            raise RuntimeError("Could not access Live application")
+        if not hasattr(app, "browser") or app.browser is None:
+            raise RuntimeError("Browser is not available in the Live application")
+
+        browser_attrs = [attr for attr in dir(app.browser) if not attr.startswith("_")]
+        path_parts = path.split("/")
+        if not path_parts:
+            raise ValueError("Invalid path")
```

Ruff (0.15.0)

[warning] Abstract `raise` to an inner function

(TRY301)